bundles/org.eclipse.passage.lic.bc/about.properties

```diff
@@ -0,0 +1,24 @@
+###############################################################################
+# Copyright (c) 2020 ArSysOp and others
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     ArSysOp - initial API and implementation
+###############################################################################
+# about.properties
+# contains externalized strings for about.ini
+# java.io.Properties file (ISO 8859-1 with "\" escapes)
+# fill-ins are supplied by about.mappings
+# This file should be translated.
+
+blurb=Passage Licensing Integration Components: Bouncy Castle integration \n\
+\n\
+Version: {featureVersion}\n\
+\n\
+(c) Copyright Eclipse contributors and others 2000.  All rights reserved.\n\
```

Comment:
Should correspond to the license header of the enclosed feature: 2018-2020 and so on

Reply:
done